frontend/src/views/project/ProjectWebhookCreate.vue

```diff
@@ -3,6 +3,19 @@
     <div class="text-xl leading-6 font-medium text-main">
       {{ $t("project.webhook.creation.title") }}
     </div>
+    <BBAttention
```

Comment:
Our project webhook (IM messages) doesn't require external URL. But it doesn't hurt to add one.

Reply:
But the backend returns error if no external url configured

Reply:
I guess we should remove this? https://sourcegraph.com/github.com/bytebase/bytebase@a2feafef34f6d6dc576a17ba5cc9950b05dd6967/-/blob/backend/api/v1/project_service.go?L674

Reply:
@RainbowDashy to confirm.

Reply:
@ecmadao  Let's remove it

Reply:
Done